change/@fluentui-react-overflow-a31beb0d-76b2-491e-add6-1f3fd27db85f.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv9 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33106&commitId=51de7816c626461437cd123d4e3384d008a0941b&env=prod&clientType=FLUENTUI)


  <details><summary><b>Drawer</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Drawer.overlay drawer full.chromium.png|1150| Changed|
|Drawer.Full Overlay High Contrast.chromium.png|2232| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>